Lib/defcon/objects/font.py

```diff
@@ -884,6 +893,8 @@ def _get_identifiers(self):
     # object as dirty.
 
     def _get_guidelines(self):
+        if self._info is None:
+            self.info
```

Comment:
> Or are guidelines only stored in self._guidelines, and only loaded from the font info?
That seems a bit odd to me.

They are, it's a legacy thing (guidelines are stored in fontinfo.plist and were previously accessed via the Info object). I mentioned that in the first post.